Sources/Clock.swift

```diff
@@ -45,12 +39,15 @@ public struct Clock {
     /// - parameter pool:       NTP pool that will be resolved into multiple NTP servers that will be used for
     ///                         the synchronization.
     /// - parameter samples:    The number of samples to be acquired from each server (default 4).
+    /// - parameter appGroupID: The shared app group your app and extensions use to share data. This will be
+    ///                         helpful if you want to use Kronos in an app extension context.
     /// - parameter completion: A closure that will be called after _all_ the NTP calls are finished.
     /// - parameter first:      A closure that will be called after the first valid date is calculated.
-    public static func sync(from pool: String = "time.apple.com", samples: Int = 4,
+    public static func sync(from pool: String = "time.apple.com", samples: Int = 4, appGroupID: String? = nil,
```

Comment:
What do you think about not adding this to here? (it doesn't really belongs IMO to the semantic of syncing a clock). I'd instead have a 

```
struct Clock {
    static let storage = TimeStorage(appGroupID: default)
}
```

so external consumers could do 
```
Clock.storage = TimeStorage(appGroupID: "something")
```

Reply:
That’s a good idea. My original hesitancy to doing that was to not make any of the storage public. But I suppose once the app group ID is public it makes sense to give it proper treatment.